openapi/src/main/java/io/micronaut/openapi/visitor/AbstractOpenApiVisitor.java

```diff
@@ -2462,4 +2460,12 @@ protected <T, A extends Annotation> List<T> processOpenApiAnnotation(Element ele
         }
         return tagList;
     }
+
+    public boolean isNullable(TypedElement element) {
```

Comment:
Please move this method to a `TypeElementUtils` class.

Reply:
done